### PR TITLE
Fix Leumi invalid password result

### PR DESCRIPTION
**Pull request by uraid**
The invalid password result was previously detected by redirection to '/InternalSite/CustomUpdate/leumi/LoginPage.ASP'.
This is not the case anymore, as an error message is presented to the user instead.
Fixed by detecting the invalid password error message.

### CODE REVIEW

File: src/scrapers/leumi.ts [hunk 1]
```diff
@@ -19,12 +19,24 @@ const FILTERED_TRANSACTIONS_URL = `${BASE_URL}/ChannelWCF/Broker.svc/ProcessRequ
...
-    [LoginResults.InvalidPassword]: [/InternalSite\/CustomUpdate\/leumi\/LoginPage.ASP/],
+    [LoginResults.InvalidPassword]: [async options => {
+      if (!options || !options.page) {
+        throw new Error('missing page options argument');
+      }
+
+      const errorMessage = await (0, _elementsInteractions.pageEvalAll)(options.page, '.errHeader', [], label => {
+        var _ref;
+
+        return (_ref = label[0]) === null || _ref === void 0 ? void 0 : _ref.innerText;
```
[the remark this answers is not in the document]
Comment (uraid) [reply src/scrapers/leumi.ts:36]: Yeah I see it now.
Changed and tested on my local machine, and edited the installed files.
Meant to copy the code from 'LoginResults.AccountBlocked' 1 line below it.

File: src/scrapers/leumi.ts [hunk 2]
```diff
@@ -25,18 +25,18 @@ const INVALID_PASSWORD_MSG = 'אחד או יותר מפרטי ההזדהות ש
...
-        return (_ref = label[0]) === null || _ref === void 0 ? void 0 : _ref.innerText;
-      });
-      return errorMessage === null || errorMessage === void 0 ? void 0 : errorMessage.startsWith(INVALID_PASSWORD_MSG);
-    }],
+    [LoginResults.InvalidPassword]: [
+      async (options) => {
+        if (!options || !options.page) {
+          throw new Error('missing page options argument');
+        }
+        const errorMessage = await pageEvalAll(options.page, '.errHeader', [], (label) => {
```
Comment (esakal) [inline src/scrapers/leumi.ts:33]: the default value should be of the same type of the expected value, so I believe it should be `''` instead of `[]`?

Comment (uraid) [reply src/scrapers/leumi.ts:33]: Sounds about right.
Also if that's the case, the 'LoginResults.AccountBlocked' case should be changed as well.
I'll edit a fix.

**Review by esakal — APPROVED** [review]

**Review by mmeir11 — APPROVED** [review]

**Comment by uraid** [comment]
It seems like CI doesn't like the defaults change to string.
How should we proceed?

**Comment by esakal** [comment]
@uraid I have learned that if you re-run the CI eventually it give up :) it looks fine now.

**Pull request merged**